h/accounts/views.py

```diff
@@ -38,6 +38,70 @@ def ajax_payload(request, data):
     return payload
 
 
+def to_xhr_response(request, non_xhr_result, form):
```

Comment:
Both this and `handle_form_submission` seem to me to be generic -- not specific to `h.accounts`. Perhaps you could put them in `h.form` and import them here?